packages/roosterjs-editor-plugins/lib/plugins/CustomReplace/CustomReplace.ts

```diff
@@ -90,14 +96,23 @@ export default class CustomReplacePlugin implements EditorPlugin {
         if (range == null) {
             return;
         }
-        const searcher = this.editor.getContentSearcherOfCursor(event);
-        const stringToSearch = searcher.getSubStringBefore(this.longestReplacementLength);
 
+        const searcher = this.editor.getContentSearcherOfCursor(event);
+        const stringToSearch = searcher
+            .getSubStringBefore(this.longestReplacementLength)
+            .replace(/\s/g, ' ');
```

Comment:
suggest move the replace call into getMAtchingReplacment(). It should be part of the matching logic.